.github/workflows/ci.yml

```
@@ -99,6 +99,10 @@ jobs:
         run: pnpm run docs:links
         background: true
 
+      - name: Check agent config drift
+        run: pnpm run agents:check
+        background: true
```

actionlint (1.7.12)

[error] unexpected key "background" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)